docs/UserGuide.md

```diff
@@ -613,44 +617,76 @@ Format: `find c [n/KEYWORD] [t/TAG]`
 
 [Back to Table of Contents](#table-of-contents)
 
-### Sort Commands
-### Sort companies based on internship date
+### Sorting contacts: `sort`
+####  Sorting companies based on internship date: `sort c`
 
-Sort all companies that have internships in the specified time period and display them in order of their next scheduled interview time (in the specified time period).
+Sort all companies that have internships in the specified time period and display them in order of their most recent scheduled interview time (in the specified time period).
 
 Format: `sort c [start/START_DATETIME] [end/END_DATETIME]`
 <div markdown="span" class="alert alert-info">
 **:pencil: Parameter Information**<br>
 * `START_DATETIME` and `END_DATETIME` must be in the format `DD-MM-YYYY HH:mm`.
 * `START_DATETIME` and `END_DATETIME` are optional and each can only be used once.
-* `START_DATETIME` must be before `END_DATETIME`.
+* `START_DATETIME` must be before `END_DATETIME`, otherwise no companies will be shown (no error message will be shown either).
```

Comment:
I think you can put the 'otherwise....' in the next line as a bullet point? Something like `- No companies or error message will be shown otherwise`